add support for destinationAmount (#81)
meso-network/meso-js: .changeset/red-candles-yawn.md

---
"@meso-network/meso-js": patch
---

> Add support for passing destinationAmount, representing the destination amount desired from the transfer. If both sourceAmount and destinationAmount are specified, destinationAmount will take precedence.

Reviewer: ```suggestion
Add support for passing `destinationAmount`, when initiating a transfer.
```

I think we can shorten this up. The rest is good detail for docs
